tc/tc_util.c

```diff
@@ -192,6 +192,86 @@ static const struct rate_suffix {
 };
 
 
+int get_percent_rate(unsigned int *rate, const char *str, char *dev)
+{
+	long value;
+	char percent[strlen(dev) - 1], r_str[20];
+	const char *pos;
+	int ret;
+	double d, r;
```

Comment:
Let's make these variables more readable: make `d` into `perc` and `r` into `rate_bytes` (or whatever the unit ends up being).